proto/cosmos/tx/v1beta1/tx.proto

```diff
@@ -105,13 +106,33 @@ message TxBody {
 
   // memo is any arbitrary note/comment to be added to the transaction.
   // WARNING: in clients, any publicly exposed text should not be called memo,
-  // but should be called `note` instead (see https://github.com/cosmos/cosmos-sdk/issues/9122).
+  // but should be called `note` instead (see
+  // https://github.com/cosmos/cosmos-sdk/issues/9122).
   string memo = 2;
 
-  // timeout is the block height after which this transaction will not
-  // be processed by the chain
+  // timeout_height is the block height after which this transaction will not
+  // be processed by the chain.
   uint64 timeout_height = 3;
 
+  // unordered, when set to true, indicates that the transaction signer(s)
+  // intend for the transaction to be evaluated and executed in an un-ordered
+  // fashion. Specifically, the account's nonce will NOT be checked or
+  // incremented, which allows for fire-and-forget as well as concurrent
+  // transaction execution.
+  //
+  // Note, when set to true, the existing 'timeout_height' value must
```

Comment:
This comment should be about `timout_timestamp` instead - needs to be reworked

will need to regen

Reply:
[lint fixes, proto comment fix](https://github.com/cosmos/cosmos-sdk/pull/23708/commits/9188d9e9d467405fba1b857eef45ca06ea168fbd)